bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/DecryptionMediator.cs

```diff
@@ -223,6 +223,10 @@ public void AccumulateShares(string tallyId, bool skipValidation = false)
     {
         return TallyDecryptions[tallyId].CreateChallenge(skipValidation);
     }
+    public void LoadChallenge(string tallyId, string guardianId, GuardianChallenge challenge)
+    {
+        TallyDecryptions[tallyId].LoadChallenge(guardianId, challenge);
```

Comment:
I think this is the spot we should be calling the accumulator

Reply:
Remember that the accumulate needs to be done once before we create these... and this only loads a single challenge so it is called multiple times.